raiden/utils/deployment.py

```diff
@@ -0,0 +1,52 @@
+import structlog
+from eth_utils import to_checksum_address
+
+from raiden.utils import get_contract_path
+from raiden.utils.solc import compile_files_cwd
+
+log = structlog.get_logger(__name__)
+
+
+# Source files for all to be deployed solidity contracts
+RAIDEN_CONTRACT_FILES = [
+    'NettingChannelLibrary.sol',
+    'ChannelManagerLibrary.sol',
+    'Registry.sol',
+    'EndpointRegistry.sol',
+]
+
+# Top level contracts to be deployed. Dependencies are handled automatically
+# in `JSONRPCClient.deploy_solidity_contract()`
+CONTRACTS_TO_DEPLOY = [
+    'Registry.sol:Registry',
+    'EndpointRegistry.sol:EndpointRegistry',
```

Comment:
what about adding the secret registry here?

Reply:
Added below this list as a separate list which would ease cleanup once the new contracts are fully integrated and old ones are removed.